integration-cli/docker_cli_tag_test.go

```diff
@@ -118,13 +118,13 @@ func (s *DockerSuite) TestTagWithSuffixHyphen(c *check.C) {
 	// test repository name begin with '-'
 	tagCmd := exec.Command(dockerBinary, "tag", "busybox:latest", "-busybox:test")
 	out, _, err := runCommandWithOutput(tagCmd)
-	if err == nil || !strings.Contains(out, "Invalid repository name (-busybox). Cannot begin or end with a hyphen") {
+	if err == nil || !strings.Contains(out, "repository name component must match") {
```

Comment:
Just noticed this test is called `TestTagWithSuffixHyphen`, but we're testing a *prefix*, not a *suffix*
(Not introduced in this PR)